recipes/openmeeg/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set version = "2.4.rc3" %}
+{% set source_tag = "2.4-rc3" %}
+
+
+package:
+  name: openmeeg
+  version: {{ version }}
+
+source:
+  url: https://github.com/openmeeg/openmeeg/archive/{{ source_tag }}.tar.gz
+  sha256: 48a52af8f9da6deef5c01d26a30ba4bdc19929999d6680dab6461b92b5a63bd7
+
+build:
+  number: 0
+  skip: true  # [win and vc<14]
+  features:
+    - vc14  # [win]
+
+requirements:
+  build:
+    - cmake >=3.8
+    - {{ compiler('cxx') }}  # [not osx]
+    - clang_osx-64  # [osx]
+    - clangxx_osx-64  # [osx]
```

Comment:
Maybe instead of this we can override the `compiler` syntax for macOS to just use `defaults` compilers. To do this, we would add a `conda_build_config.yaml` file next to the `meta.yaml`, which specified something like the following (may need tweaks as I haven't tested this yet).

```yaml
c_compiler:         # [osx]
  - clang_osx-64    # [osx]
cxx_compiler:       # [osx]
  - clangxx_osx-64  # [osx]
```

Reply:
Does this make sense?

Reply:
Updated

Reply:
I did not manage to make it work. (see https://travis-ci.org/conda-forge/staged-recipes/builds/396310460#L339) Not even after reading [this](
https://github.com/conda/conda-docs/blob/master/docs/source/user-guide/tasks/build-packages/compiler-tools.rst)

Reply:
I was planning to address everything and push it back again so that we could see the difference in the CI.

Reply:
(sorry I did not see the updated)

Reply:
Is there a reason why the category should also be tagged `# [osx]`

Reply:
Or why not this

Reply:
Oops, it should be

```yaml
c_compiler:         # [osx]
  - clang_osx-64    # [osx]
cxx_compiler:       # [osx]
  - clangxx_osx-64  # [osx]
```

Sorry I mistyped earlier.

Reply:
Oops, I guess I mistyped earlier. Do you mind explaining this further, @isuruf?

Reply:
@jakirkham, that's wrong

Reply:
Sure, so I saw you edited my comment. ;) Care to explain why this update is correct? :)

Reply:
That's how `compiler('cxx')` works. It expands to `{{ cxx_compiler }}_{{ target_platform }}`